mypy/checkexpr.py

```diff
@@ -3968,54 +3968,14 @@ def check_op_reversible(
         right_expr: Expression,
         context: Context,
     ) -> tuple[Type, Type]:
-        def lookup_operator(op_name: str, base_type: Type) -> Type | None:
```

Comment:
Extracting this helper to method is good and may improve performance of compiled mypy - `mypyc` isn't doing its best when there are nested functions.